extra/macos_pkg_files/license.txt

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
GNU license is already included in root of project. Doesn't need to be copied over to this directory.

Reply:
Hey, thanks for the review!
Packages didn't allow me to choose the license file at the root, so I had to include a txt copy for the installer, that's why it's there

Reply:
Nvm, modifying the `pkgproj` directly lets me use  the one in the root, removing this right now...